src/utils/misc.ts

```diff
@@ -5,3 +5,35 @@ export const getNumberId = (data: any[], idField: string) => {
 export const delay = (ms: number) => {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
+
+export const getResponse = (
```

Comment:
let's move this to `object-utils`(create if we don't have such a thing). Also, let's rename it to `sepPropDeep(obj, prop, value)`.